components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -0,0 +1,258 @@
+      expenseStats(accounts: $accounts, from: $dateFrom, to: $dateTo) {
+        numExpenses
+        dailyAverage
+        numInvoices
+        numReimbursements
+        numGrants
+      }
+    }
+  }
+`;
+
+const TransactionsOverviewSection = ({ hostSlug, currency }) => {
+  const [dateFrom, setDateFrom] = useState(null);
+  const [dateTo, setDateTo] = useState(null);
+  const [collectives, setCollectives] = useState(null);
+
+  const { data, loading, refetch } = useQuery(transactionsOverviewQuery, {
+    variables: { hostSlug, dateFrom, dateTo, accounts: collectives },
+    context: API_V2_CONTEXT,
+  });
+  const contributionStats = data?.host.contributionStats;
+  const expenseStats = data?.host.expenseStats;
+
+  useEffect(() => {
+    refetch();
+  }, [dateFrom, dateTo, collectives]);
+
+  const { numContributions, numRecurring, numOneTime, dailyAvgIncome } = contributionStats || 0;
+  const { numExpenses, numInvoices, numReimbursements, numGrants, dailyAverage } = expenseStats || 0;
+
+  const setDate = dateRange => {
+    if (!dateRange) {
+      setDateFrom(null);
+      setDateTo(null);
+      refetch();
+      return;
+    }
+    const dates = dateRange.split('→');
+    const dateFrom = dates[0];
+    const dateTo = dates[1];
+    if (dateFrom === 'all') {
+      setDateFrom(null);
+    } else {
+      setDateFrom(dateFrom);
+    }
+    if (dateTo === 'all') {
+      setDateTo(null);
+    } else {
+      setDateTo(dateTo);
+    }
+  };
+
+  const getDateString = () => {
+    let dateString;
+    if (!dateFrom && !dateTo) {
+      return;
+    }
+    if (dateFrom) {
+      dateString = `${dateFrom}→`;
+    } else {
+      dateString = 'all→';
+    }
+
+    if (dateTo) {
+      dateString += dateTo;
+    } else {
+      dateString += 'all';
+    }
+    return dateString;
+  };
+
+  const setCollectiveFilter = collectives => {
+    if (collectives.length === 0) {
+      setCollectives(null);
+    } else {
+      const collectiveIds = collectives.map(collective => ({ legacyId: collective.value.id }));
+      setCollectives(collectiveIds);
+    }
+  };
+
+  return (
+    <React.Fragment>
+      <Flex flexWrap="wrap">
+        <Container width={[1, 1, 1 / 2]} pr={2} mb={[3, 3, 0, 0]}>
+          <FilterLabel htmlFor="transactions-period-filter">
+            <FormattedMessage id="TransactionsOverviewSection.PeriodFilter" defaultMessage="Filter by Date" />
+          </FilterLabel>
+          <PeriodFilter onChange={value => setDate(value)} value={getDateString()} />
+        </Container>
+        <Container width={[1, 1, 1 / 2]}>
+          <FilterLabel htmlFor="transactions-collective-filter">
+            <FormattedMessage id="TransactionsOverviewSection.CollectiveFilter" defaultMessage="Filter by Collective" />
+          </FilterLabel>
+          <CollectivePickerAsync
+            inputId="TransactionsCollectiveFilter"
+            data-cy="transactions-collective-filter"
+            types={['COLLECTIVE']}
+            isMulti
```

Comment:
We only care about accounts under this host

```suggestion
            hostCollectiveIds={host.legacyId}
```